app/locales/en.json

```diff
@@ -164,6 +164,7 @@
     "report_symptoms_title": "How you feel?",
     "see_exposure_history": "See exposure history",
     "settings_title": "Dashboard",
+    "terms_and_conditions": "Terms & Conditions",
```

Comment:
Please follow the order convention in this file.

Reply:
Which one?